src/components/footer/footer.js

```diff
@@ -18,20 +20,40 @@ const Footer = () => {
 
   return (
     <footer className="custom-footer">
+      <div className="custom-footer__logo">
```

Comment:
Redundant div. If you want to customize Logo with styles, pass className prop directly to Logo

Reply:
I need this div for logo position

Reply:
But I delete className for component Logo